### PR TITLE
Guide Creation Rejig

### PR DESCRIPTION
**Pull request by simonv3**
Moved some things around so that the new guide creation flow is more in line with the current designs. 

Everything works locally. The issue, as always, is on heroku. 
screenshot 2015-10-29 20 09 04

### CODE REVIEW

File: app/assets/javascripts/angular-libs/guides/new/guides.new.guideStageOverview.directive.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+              image_url: image
+            });
+          };
+
+          $scope.$watch('viewingStageOverview', function() {
+
+          });
+        }
+      ],
+      templateUrl: '/assets/angular-libs/guides/new/guides.new.guideStageOverview.template.html'
```
Comment (simonv3) [inline app/assets/javascripts/angular-libs/guides/new/guides.new.guideStageOverview.directive.js]: Tough pickles, hound.

**Comment by roryaronson** [comment]
Yay! Let me know when this is on staging and I can test stuff out :)

**Comment by simonv3** [comment]
It is now! Warning, it's very minimalist, I didn't add any new features (eg timeline). And what's not working on Heroku is most likely image uploading, cause I still haven't figured out a good way to debug that.

**Comment by roryaronson** [comment]
Yay! Everything worked for me :) (except photos)

It already feels a lot smoother and more logical than what we currently have.

**Pull request merged**